role_test.go

```diff
@@ -0,0 +1,56 @@
+package jira
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+)
+
+func TestRoleService_GetList(t *testing.T) {
+	setup()
+	defer teardown()
+	testAPIEndpoint := "/rest/api/3/role"
+
+	raw, err := ioutil.ReadFile("./mocks/all_roles.json")
+	if err != nil {
+		t.Error(err.Error())
+	}
+	testMux.HandleFunc(testAPIEndpoint, func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testRequestURL(t, r, testAPIEndpoint)
+		fmt.Fprintf(w, string(raw))
+	})
+
+	roles, _, err := testClient.Role.GetList()
+	if roles == nil {
+		t.Error("Expected role list. Role list is nil")
+	}
+	if err != nil {
+		t.Errorf("Error given: %v", err)
+	}
+}
+
+func TestRoleService_Get(t *testing.T) {
```

Comment:
Please add negative tests (those that validate when an error should be returned)